infrastructure/staging/main.tf

```diff
@@ -218,3 +218,52 @@ module "worker_node_1" {
 
   worker_node_container_image = local.worker_node_container_image
 }
+
+module "worker_node_1_autoscaling_cloudwatch" {
```

Comment:
We have modules to do autoscaling already: https://github.com/wellcomecollection/catalogue/blob/master/infrastructure/modules/worker/main.tf#L33

If those work for you i'd prefer we re-use them, but i'm also happy if you find the `umotif-public` one easier to work with.

Reply:
The `umotif-public` autoscaling module allows us to use custom metrics with the `metric_query`. As we wanted to use the number of visible messages in the job queue for scaling out and back in, I used the `umotif-public` module.